app/src/ui/dispatcher/error-handlers.ts

```diff
@@ -462,3 +463,65 @@ export async function localChangesOverwrittenHandler(
 
   return null
 }
+
+const rejectedPathRe = /^ ! \[remote rejected\] .*? -> .*? \(refusing to allow an integration to create or update (.*?)\)$/m
```

Comment:
Yeah, we used to have three distinct error messages from dotcom depending on whether the change was an addition, deletion, or modification. These three have since been coalesced into the "create or update" terminology.